### ISSUE TITLE
Serverless Reading Hardcoded AWS Credentials File on Windows

### ISSUE THREAD

**Issue by beittenc** [serverless/serverless#5866]
# This is a Bug Report

## Description
Under Windows, Serverless appears to be hardcoded to read the AWS credentials file in /c/Users/<profile>/.aws/credentials, ignoring what the user's HOME directory is actually set to. This is inconsistent with the rest of the AWS command line tools and produces a very confusing experience.

For reasons, my HOME directory is set to my OneDrive directory like this: /c/Users/<profile>/OneDrive - <org name>. This means that my ~/.aws/credentials file resides within OneDrive and not in the default /c/Users/<profile>/.aws/ location. The regular AWS command line tools as well as other tools such as aws-mfa understand and obey this properly. Serverless, however, is always looking in /c/Users/<profile> and then spits out errors about my profile being missing or my security token being expired.

Similar or dependent issues:
* #5855 - This is not exactly the same problem, but they are similar in the idea of "mistreating AWS credentials files"
* #5039 

* ***Serverless Framework Version you're using***: 1.36.1
* ***Operating System***: Windows 10

**Comment by dschep** [comment]
just to double check, do you mean `/c/Users/USERNAMEHERE/.aws/credentials` and `/c/Users/USERNAMEHERE/OneDrive`?

**Comment by beittenc** [comment]
Yes, sorry. I didn't check to see that Github markup was eating my syntax. I'll see if I can fix.

**Comment by dschep** [comment]
Cool thanks. I'll have to check this out next time I spin up my windows VM.
